src/TodoList.js

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+const todoList = [
+    {
+      id: 1,
+      title: "Complete assignment"
+    },
+    {
+      id: 2,
+      title: "Create branch"
+    },
+    {
+      id: 3,
+      title: "Push to GitHub"
+    },
+    {
+      id: 4,
+      title: "Create pull request"
+    }
+  ];
+function TodoList() {
+    return (<ul>
```

Comment:
nit:(nit-picky) I would start the opening `ul` element on a new line. This way the opening/ closing tag line up in the IDE. Makes it a little easier to read.